src/site/markdown/index.md

```diff
@@ -1,3 +1,48 @@
-# Documentation
+# TableJoin Java Client
+The TableJoin utility is designed for joining tables expressed in CSV or structed as list of String lists. 
+It enhances the flexibility of data processing by allowing developers to perform joint operations on specified tables.
 
-This is the documentation.
+# How to run the TableJoin Client inside Java
+
+Add TableJoin in the list of dependencies in your [Maven **pom.xml**](https://maven.apache.org/pom.ht):
+
+```
+<dependencies>
+	<!-- [...] -->
+	<dependency>
+		<groupId>org.sentrysoftware</groupId>
```

Comment:
`org.sentrysoftware` could be replaced with `${project.groupId}`, and `tablejoin` with `${project.artifactId}`.

So that if we change the coordinates, this will be updated automatically ;-)